MediaBrowser.Controller/SystemBackupService/BackupManifestDto.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+
+namespace MediaBrowser.Controller.SystemBackupService;
+
+/// <summary>
+/// Manifest type for backups internal structure.
+/// </summary>
+public class BackupManifestDto
```

Comment:
Maybe add the path here too, so a frontend actually knows the paths it can restore from